test/e2e/run_test.sh

```diff
@@ -283,9 +375,17 @@ if [[ -z ${SKIP_SETUP:-} ]]; then
 	setup_kmesh
 fi
 
+setup_kmesh_log
+
+capture_pod_logs &
+
 cmd="go test -v -tags=integ $ROOT_DIR/test/e2e/... -istio.test.kube.loadbalancer=false ${PARAMS[*]}"
 
-bash -c "$cmd"
+bash -c "$cmd" || {
+	cat $LOGFILE
+	rm $LOGFILE
+	exit 1
```

Comment:
why exit without cleaning up. I think failure does not mean not cleaning up, they are orthogonal